apps/prairielearn/src/pages/instructorAssessmentManualGrading/instanceQuestion/rubricSettingsModal.html.ts

```diff
@@ -15,7 +15,11 @@ export function RubricSettingsModal({ resLocals }: { resLocals: Record<string, a
         >
           <input type="hidden" name="__csrf_token" value="${resLocals.__csrf_token}" />
           <input type="hidden" name="__action" value="modify_rubric_settings" />
-          <input type="hidden" name="modified_at" value="${rubric_data?.modified_at.toString()}" />
+          <input
```

Comment:
rubric_data is parsed on the server by Zod in `selectRubricData`, and this is a server side page, so we can expect this is a string.